src/jaxsim/rbda/contacts/relaxed_rigid.py

```diff
@@ -333,10 +343,22 @@ def compute_contact_forces(
         # Compute the position in the constraint frame.
         position_constraint = jax.vmap(lambda δ, n̂: -δ * n̂)(δ, n̂)
 
+        # Compute the regularization terms.
+        a_ref, r, *_ = self._regularizers(
+            model=model,
+            position_constraint=position_constraint,
+            velocity_constraint=velocity,
+            parameters=model.contact_params,
+        )
+
         # Compute the transforms of the implicit frames corresponding to the
         # collidable points.
         W_H_C = js.contact.transforms(model=model, data=data)
 
+        # Retrieve the kinematic constraints, if any.
+        kin_constraints: ConstraintMap = model.kin_dyn_parameters.constraints
+        n_kin_constraints: int = 6 * kin_constraints.frame_idxs_1.shape[0]
```

Comment:
Is the type hint necessary here?
```suggestion
        kin_constraints = model.kin_dyn_parameters.constraints
        n_kin_constraints = 6 * kin_constraints.frame_idxs_1.shape[0]
```

Reply:
Since `kin_dyn_parameters` is optional, my IDE is not able to identify the type of its `constraints` attribute, hence in this way it's easier for it to detect the type in the following of the function. Can I leave it like this?

Reply:
Yes of course! It was only to understand if there were specific issues related to this